includes/class-wc-admin-reports-sync.php

```diff
@@ -432,12 +432,31 @@ public static function orders_lookup_import_batch( $batch_number, $days, $skip_e
 	 * @return void
 	 */
 	public static function orders_lookup_import_order( $order_id ) {
+
+		$order = wc_get_order( $order_id );
+		$type  = $order->get_type();
```

Comment:
This shouldn't be called before the check that it's a valid order.

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-admin/pull/2425/commits/810088e2632744f369435aed0621a1db848b77df